easybuild/easyconfigs/l/libxc/libxc-6.2.2-GCC-12.3.0.eb

```diff
@@ -0,0 +1,45 @@
+easyblock = 'CMakeMake'
+
+name = 'libxc'
+version = '6.2.2'
+
+homepage = 'https://www.tddft.org/programs/libxc'
+description = """Libxc is a library of exchange-correlation functionals for density-functional theory.
+ The aim is to provide a portable, well tested and reliable set of exchange and correlation functionals."""
+
+toolchain = {'name': 'GCC', 'version': '12.3.0'}
+
+source_urls = ['https://www.tddft.org/programs/libxc/down.php?file=%(version)s/']
+sources = [SOURCE_TAR_GZ]
+checksums = ['a0f6f1bba7ba5c0c85b2bfe65aca1591025f509a7f11471b4cd651a79491b045']
+
+builddependencies = [
+    ('CMake', '3.26.3'),
+    ('Perl', '5.36.1'),
+]
+
+local_common_configopts = "-DENABLE_FORTRAN=ON -DENABLE_FORTRAN03=ON -DENABLE_XHOST=OFF"
```

Comment:
```suggestion
local_common_configopts = "-DENABLE_FORTRAN=ON -DENABLE_XHOST=OFF "

# don't disable building of third and fourth derivates, since it's required by some software that depends on libxc
# (like ABINIT, which requires "3rd derivatives of energy")
# see also https://github.com/pyscf/pyscf/issues/1103
local_common_configopts += "-DDISABLE_KXC=OFF -DDISABLE_LXC=OFF"
```